### ISSUE TITLE
schemas support

### ISSUE THREAD

**Issue by josejachuf** [gi0baro/weppy#211]
Hello @gi0baro . In which state is the schemas support (pgsql)?
It would be desirable to have it
Jose

**Comment by gi0baro** [comment]
@josejachuf I agree with you, but I think it's almost impossible on 1.x series due to the lack of support in pydal. Would be achievable with 2.x series with the ORM rewrite.
